GETTING-STARTED.md

```diff
@@ -21,11 +21,11 @@ When writing examples:
 
 ## Testing
 
-PatternFly React currently uses [Jest](https://facebook.github.io/jest/) for running snapshot tests and [Cypress](https://www.cypress.io/) for running integration tests.
+PatternFly React currently uses [Jest](https://facebook.github.io/jest/) with [React Testing Library](https://github.com/testing-library/react-testing-library) for running unit tests and [Cypress](https://www.cypress.io/) for running integration tests.
```

Comment:
```suggestion
PatternFly React currently uses [Jest](https://facebook.github.io/jest/) with [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/) for running unit tests and [Cypress](https://www.cypress.io/) for running integration tests.
```

Reply:
Yeah, I went back and forth about which would be better to link here, I totally get leaning towards linking their docs directly rather than their repo.